sms/v1/openapi.yaml

```diff
@@ -69,8 +69,6 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/RestMessageResponse"
-        "201":
-          description: Created
```

Comment:
These operations return 200, so 201 can be deleted I guess. Am I right?

Reply:
Yes you are right